modules/web-console/web-console-server/src/main/resources/ignite-config.xml

```diff
@@ -22,82 +22,15 @@
-                <property name="targetMethod" value="getProperties"/>
-            </bean>
-        </property>
-        
-        <property name="targetMethod" value="putAll"/>
-        <property name="arguments">
-            <util:properties>
-                <prop key="IGNITE_JETTY_PORT">60080</prop>
-            </util:properties>
-        </property>
-    </bean>
-
     <bean class="org.apache.ignite.configuration.IgniteConfiguration">
-        <property name="authenticationEnabled" value="false"/>
         <property name="metricsLogFrequency" value="0"/>
-
         <property name="queryThreadPoolSize" value="16"/>
         <property name="failureDetectionTimeout" value="10000"/>
-        <property name="clientFailureDetectionTimeout" value="10000"/>
         <property name="networkTimeout" value="10000"/>
 
-        <property name="cacheConfiguration">
-            <bean class="org.apache.ignite.configuration.CacheConfiguration">
-                <property name="name" value="*"/>
-                <property name="cacheMode" value="PARTITIONED"/>
-                <property name="backups" value="1"/>
-
-                <property name="affinity">
-                    <bean class="org.apache.ignite.cache.affinity.rendezvous.RendezvousAffinityFunction">
-                        <property name="partitions" value="256"/>
-                    </bean>
-                </property>
-            </bean>
-        </property>
-
-        <property name="connectorConfiguration">
-            <bean class="org.apache.ignite.configuration.ConnectorConfiguration">
-                <property name="port" value="60300"/>
-            </bean>
-        </property>
-
+        <property name="connectorConfiguration"><null/></property>
         <property name="clientConnectorConfiguration"><null/></property>
 
-        <property name="communicationSpi">
-            <bean class="org.apache.ignite.spi.communication.tcp.TcpCommunicationSpi">
-                <property name="localPort" value="60200"/>
-                <!-- Sets socket write timeout for TCP connection. -->
-                <property name="socketWriteTimeout" value="10000"/>
-            </bean>
-        </property>
-
-        <!-- Allows encryption with SSL and TLS. For more details refer to https://apacheignite.readme.io/docs/ssltls -->
-        <!--
-            <property name="sslContextFactory">
-              <bean class="org.apache.ignite.ssl.SslContextFactory">
-                <property name="keyStoreFilePath" value="keystore/server.jks"/>
-                <property name="keyStorePassword" value="123456"/>
-                <property name="trustStoreFilePath" value="keystore/trust.jks"/>
-                <property name="trustStorePassword" value="123456"/>
-                <property name="setProtocol" value="SSL"/>
-              </bean>
-            </property>
-        -->
-
-        <property name="transactionConfiguration">
```

Comment:
Why we remove `transactionConfiguration`?

Reply:
Fixed